streams/src/main/java/org/apache/kafka/streams/kstream/internals/GroupedStreamAggregateBuilder.java

```diff
@@ -110,11 +115,8 @@
     /**
      * @return the new sourceName if repartitioned. Otherwise the name of this stream
      */
-    private String repartitionIfRequired(final String repartitionTopicNamePrefix,
-                                         final OptimizableRepartitionNode.OptimizableRepartitionNodeBuilder<K, V> optimizableRepartitionNodeBuilder) {
-        if (!repartitionRequired) {
-            return this.name;
-        }
+    private String createRepartitionSource(final String repartitionTopicNamePrefix,
```

Comment:
I did not in-line this method as I feel it helps with the readability of the code.

Reply:
Does it? It's a one liner calling `createRepartitionedSource`, so what does we gain by calling `createRepartitionSource` instead?

Reply:
I guess it's personal preference if you insist I can inline it.

Reply:
Don't insist -- would prefer though (for this case) -- in general, this pattern can be useful -- just don't see it for this particular case.